<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_50">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_100">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_500">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Coffee">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Tea">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Milk">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_reset">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
